explorer/ast/bindings.cpp

```diff
@@ -37,21 +37,33 @@ void Bindings::Add(Nonnull<const GenericBinding*> binding,
   }
 }
 
-auto Bindings::None() -> Nonnull<const Bindings*> {
-  static Nonnull<const Bindings*> bindings = new Bindings;
-  return bindings;
-}
-
 void Bindings::Print(llvm::raw_ostream& out) const {
-  out << "{";
+  std::vector<std::pair<Nonnull<const GenericBinding*>, Nonnull<const Value*>>>
+      args;
+
+  for (const auto& [binding, value] : args_) {
+    args.push_back({binding, value});
+  }
```

Comment:
You can construct the vector directly from the map because the elements are the same type:
```suggestion
  std::vector<std::pair<Nonnull<const GenericBinding*>, Nonnull<const Value*>>>
      args(args_.begin(), args_.end());
```